test/distributed_exec.jl

```diff
@@ -1927,6 +1927,21 @@ begin
     end
 end
 
+# test logging
+w = only(addprocs(1))
+@everywhere using Logging
+@test_logs (:info, "from pid $w") begin
+    prev_logger = global_logger(current_logger())
+    try
+        wait(@spawnat w with_logger(RemoteLogger(1)) do
+                @info("from pid $(myid())")
+            end)
```

Comment:
```suggestion
        wait(@spawnat w with_logger(RemoteLogger(1)) do
                @info("from pid $(myid())")
            end)
```
Can we test that `"from pid $(myid())"` was even logged?

Reply:
I will look into it, maybe we can catch it with another logger and see if it's captured.

Reply:
Right, it's actually captured by `@test_logs` which encloses the whole block.